synthetic_data/marginal_dist.py

```diff
@@ -17,26 +17,28 @@ def _gen_rv_hist_continuous(col_stats):
         dist (dict): Dictionary stating distribution type along with other parameters
         for the distribution.
     """
-
     bin_counts, bin_edges = (
         col_stats["histogram"]["bin_counts"],
         col_stats["histogram"]["bin_edges"],
     )
 
     # Create a continuous distribution from the histogram and sample data from it
     hist_dist = stats.rv_histogram((bin_counts, bin_edges))
-    observed_samples = hist_dist.rvs(size=1000)
+    # observed_samples = hist_dist.rvs(size=1000)
```

Comment:
why is this getting removed?

Reply:
flake8 was complaining that it assigned but never used or returned

Reply:
interesting...

Reply:
that is odd... add a comment above just so folks know what's going on with this. That's a fairly large change and I'd prefer to remove entirely

Reply:
It's a relic and should be removed. @drahc1R is correct here.